hepa-uv/firmware/hepauv_hardware.c

```diff
@@ -1,16 +1,18 @@
-#include "hepa-uv/firmware/led_hardware.h"
+#include "hepa-uv/firmware/hepauv_hardware.h"
```

Comment:
i think if you want to merge things like this you should instead restructure into timer_hardware and button_hardware or something